services/brig/deb/opt/brig/templates/de/user/email/invitation.html

```diff
@@ -1,102 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Note that invitations have been deprecated and will be removed; this is why the German version was not ported over but the English was; we still need it, otherwise `brig` will not start. I'd suggest, to eliminate confusion, to fully remove it from `brig` (less is more!)

Reply:
Removing is my middle name! Just say when :)